routing/redirect_trailing_slash.rst

```diff
@@ -27,7 +27,7 @@ new URL with a 301 response status code::
 
             $url = str_replace($pathInfo, rtrim($pathInfo, ' /'), $requestUri);
 
-            return $this->redirect($url, 301);
+            return $this->redirect($url, 308);
```

Comment:
Should not be 301 if it's a `GET` method?

Reply:
No, why?

Reply:
See https://httpstatuses.com/308, it works for both `GET` and `POST`, and forbids changing them.

Reply:
:+1: Then. But not all people know what `308` code mean. It would be great to add an explanation line with the link you just provided.